### PR TITLE
fix: add error handling for failed file upload on push command

### PR DESCRIPTION
**Pull request by jacobator**
fix https://github.com/Redocly/openapi-cli/issues/248

### CODE REVIEW

**Review by Marshevskyy — COMMENTED** [review]
push command does not have try/catch. I think it would be good to add

**Comment by Marshevskyy** [comment]
Also it would be great to add at least some unit tests

File: packages/cli/src/commands/push.ts [hunk 1]
```diff
@@ -217,15 +224,17 @@ function getDestinationProps(destination: string) {
...
+  const response = await fetch(url, {
     method: 'PUT',
     headers: {
       'Content-Length': fileSizeInBytes.toString()
     },
     body: readStream
-  })
+  });
+
+  return response.ok;
```
Comment (Marshevskyy) [inline packages/cli/src/commands/push.ts:239]: return just response

File: packages/cli/src/commands/push.ts [hunk 2]
```diff
@@ -224,17 +224,15 @@ function getDestinationProps(destination: string) {
   return destination.substring(1).split(/[@\/]/);
 }
 
-async function uploadFileToS3(url: string, filePathOrBuffer: string | Buffer): Promise<boolean> {
+function uploadFileToS3(url: string, filePathOrBuffer: string | Buffer) {
```
Comment (Marshevskyy) [inline packages/cli/src/commands/push.ts]: why did you remove async?

Comment (jacobator) [reply packages/cli/src/commands/push.ts]: because I removed await inside the function

**Review by Marshevskyy — APPROVED** [review]
good job on tests!
